src/main/java/nl/tudelft/contextproject/model/Inventory.java

```diff
@@ -57,6 +62,7 @@ public void remove(Entity ent) {
 				}
```

Comment:
Not relevant for this PR, but I see here that an entity is removed in a for each loop over it, which WILL throw a ConcurrentModificationException.

I will create a PR to fix this after this has been merged (to avoid merge conflicts).